v3/src/components/graph/v2-graph-exporter.ts

```diff
@@ -230,13 +232,39 @@ function getPlotModels(graph: IGraphContentModel): Partial<ICodapV2GraphStorage>
   return storage
 }
 
+const getTransparency = (color: string, type: "point" | "stroke") => {
+  const rgbaColor = colord(color).toRgb()
+  return rgbaColor.a ?? (type === "point" ? 0.84 : 1)
```

Comment:
I moved this function here because it contains graph-specific logic rather than something appropriate to all clients of `color-utils`. That said, in practice it doesn't matter because `toRgb()` is documented to always return an alpha, so the `?? (type === "point" ? 0.84 : 1)` will never have any effect. If there are some circumstances in which `0.84` should be returned, then the logic will need to be clarified.